src/app/components/modal-create-tile-server/modal-create-tile-server.component.html

```diff
@@ -0,0 +1,193 @@
+      </div>
+      <div class="small-6 cell">
+        <label> Attribution Link:
+          <input type="url"
+                 pattern="https?://.+"
+                 formControlName="attributionLink"
+                 required="false">
+        </label>
+      </div>
+    </div>
+    <label *ngIf="tsCreateForm.get('type').value == 'tms'"> Zoom Max:
+      <input type="number"
+             formControlName="maxZoom"
+             required="false">
+    </label>
+    <label *ngIf="tsCreateForm.get('type').value == 'tms'"> Zoom Min:
+      <input type="number"
+             formControlName="minZoom"
+             required="false">
+    </label>
+    <label *ngIf="tsCreateForm.get('type').value == 'wms'"> Layers (comma-separated):
+      <input type="text"
+             formControlName="layers"
+             required="false">
+    </label>
+    <label *ngIf="tsCreateForm.get('type').value == 'wms'"> Params:
+      <input type="text"
+             formControlName="params"
+             required="false">
+    </label>
+    <label *ngIf="tsCreateForm.get('type').value == 'wms'"> Format:
+      <select formControlName="format" required="false">
+        <option value="image/png">png</option>
+        <option value="image/jpeg">jpeg</option>
+      </select>
+    </label>
+  </div>
+
+  <div *ngIf="tsCreateForm.get('method').value == 'ini'">
+    <app-file-browser
+      [allowFolders]="false"
+      [allowedExtensions]="tapisFilesService.IMPORTABLE_TILE_TYPES"
+      [single]="true"
+      [helpText]=null
+      [heading]=null
+      (selection)="onDSFileSelection($event)"
+    >
+    </app-file-browser>
+  </div>
+
+  <div *ngIf="tsCreateForm.get('method').value == 'qms'">
+    <div class="grid-x">
+      <input #qmsSearch
+             class="cell auto qms-search-input"
+             name=""
+             type="text"
+             value="" />
+      <button class="button small cell large-1 qms-search-button"
+              (click)="searchQMS($event, qmsSearch.value)">
+        <i class="fas fa-search"></i>
+      </button>
+    </div>
+    <div class="qms-options">
+      <label> Server Type:
+        <select formControlName="type" required="false">
+          <option value="">All</option>
+          <option value="tms">TMS</option>
+          <option value="wms">WMS</option>
+        </select>
+      </label>
+
+      <label> Order By:
+        <select formControlName="ordering" required="false">
+          <option value="name">Name</option>
+          <option value="id">ID</option>
+          <option value="created_at">Created At</option>
+          <option value="updated_at">Updated At</option>
+        </select>
+      </label>
+
+      <label> Order:
+        <select formControlName="order" required="false">
+          <option value="">Ascending</option>
+          <option value="-">Descending</option>
+        </select>
+      </label>
+
+    </div>
+    <div class="qms-search-results">
+      <div *ngFor="let qs of qmsSearchResults">
```

Comment:
I feel like either here there should be some notification that that the search is being performed and (new) results are about to show up.  Maybe a spinner here or alternatively by the magnifying glass?  what do you think?